docs/how-to-guides/meson-args.rst

```diff
@@ -17,5 +16,98 @@ Passing arguments to Meson_
+
+   - meson dist: https://mesonbuild.com/Commands.html#dist
+   - meson setup: https://mesonbuild.com/Commands.html#setup
+   - meson compile: https://mesonbuild.com/Commands.html#compile
+   - meson install: https://mesonbuild.com/Commands.html#install
+
+These arguments can be added to the project via the project's ``pyproject.toml``:
+
+.. code-block:: console
+
+   [tool.meson-python.args]
+   dist = ['dist-argument_1','dist-argument_2', '...']]
+   setup = ['setup-argument_1','setup-argument_2', '...']]
+   compile = ['compile-argument_1','compile-argument_2', '...']]
+   install = ['install-argument_1','install-argument_2', '...']]
+
+.. admonition:: All four categories must exist
+   :class: attention
+
+   Currently all four option categories (dist, setup, compile and install) must be present in the [tool.meson-python.args] section, even if no arguments are supplied (empty list: [])
+
+Or using the build config settings:
+
+.. code-block:: console
+
+   # with pypa/build
+   python -m build -Cdist-args="args"    \
+                   -Csetup-args="args"   \
+                   -Ccompile-args="args" \
+                   -Cinstall-args="args" .
+
+   # with pip
+   python -m pip --config-settings=dist-args="args"    \
+                 --config-settings=setup-args="args"   \
+                 --config-settings=compile-args="args" \
+                 --config-settings=install-args="args" .
+
+
+Example 1: set the default libraries to "static"
+================================================
+
+Set the target default libraries to "static" in the pyproject.toml
+
+.. code-block:: console
+
+   [tool.meson-python.args]
+   dist = []
+   setup = ["--default-library=static"]
+   compile = []
+   install = []
+
+Or using build config settings:
+
+.. code-block:: console
+
+   # with pypa/build
+   python -m build -Csetup-args="--default-library=static" .
+
+   # with pip
+   python -m pip --config-settings=setup-args="--default-library=static" .
+
+
+Example 2: use meson install_tags for selective installs
+========================================================
+
+It is now (meson-python >= 0.13) possible to use meson install_tags to select which targets are installed into the binary wheels.
+
+.. admonition:: Meson installation tags
+   :class: seealso
+
+   Each meson target has a default install_tag (e.g. 'runtime' for shared libraries and 'devel' for headers.). Using "meson install --tags=tag1,tag2,..." will cause meson to only install the targets tagges with any of the specified tags. The default tag of each target can be overwritten using the target's "install_tag" option. Ffor more information refer mesons documentation in installation-tags: https://mesonbuild.com/Installing.html#installation-tags
+
+The following code (in pyproject.toml) causes meson-python to only install targets tagged with 'runtime' or 'python-runtime') into the binary wheel (ignoring e.g. c++ headers):
```

Comment:
`runtime` and `python-runtime` should be wrapped in `` instead.

> c++ headers

The C in C++ should be capitalized.